xtokens/src/lib.rs

```diff
@@ -245,7 +330,7 @@ pub mod module {
 			let weight = T::Weigher::weight(&mut msg).map_err(|()| Error::<T>::UnweighableMessage)?;
 			T::XcmExecutor::execute_xcm_in_credit(origin_location, msg, weight, weight)
 				.ensure_complete()
-				.map_err(|_| Error::<T>::XcmExecutionFailed)?;
+				.map_err(xcm_to_pallet_error::<T>)?;
```

Comment:
I think it is enough to print the log here.

Reply:
may be. 
may be you know how to get logs in xcm simulator tests https://stackoverflow.com/questions/70085232/how-to-see-logs-out-of-tests-running-substrate-cumulus-based-runtime ?

Reply:
1. add `env_logger::init();` in the top of test method.
2. Run with
```
RUST_LOG=trace,parity-db=error,trie=error cargo test
```